incubator-tools/docai_processor_migration/docai_processor_migration.ipynb

```diff
@@ -0,0 +1,960 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #73.        url = get_base_url(destination_processor_name) + "/dataset:importDocuments"
</pre><p>This function should use the client libraries instead of calling the REST url manually. Same for all other functions in this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>